cdmtaskservice/models.py

```diff
@@ -18,6 +19,15 @@
 _PATH_REGEX=r"^[\w.-/]+$"
 
 
+def _err_on_control_chars(s: str, allowed_chars: list[str]):
```

Comment:
```suggestion
def _err_on_control_chars(s: str, allowed_chars: list[str] = None):
```

Reply:
Hmm, I'm surprised my manual tests passed. Fixed locally, it'll be in the next PR.